sa/storage-authority.go

```diff
@@ -31,6 +33,114 @@ func digest256(data []byte) []byte {
 	return d.Sum(nil)
 }
 
+var dialectMap map[string]interface{} = map[string]interface{}{
+	"sqlite3":  gorp.SqliteDialect{},
+	"mysql":    gorp.MySQLDialect{Engine: "InnoDB", Encoding: "UTF8"},
+	"postgres": gorp.PostgresDialect{},
+}
+
+// Models
+type Registration struct {
+	Thumbprint        string `db:"thumbprint"`
+	core.Registration
+
+	LockCol int64
+}
+
+type Pending_auth struct {
```

Comment:
Ah, I see, it's to match the existing table name. We can rename the table to match the others.

Also note that the z isn't for pluralization: The shorthand `authz` is used for `authorization`, and opposed to `authn` which is shorthand for `authentication`. It's a terrible shorthand since they but end in `n`, but :shrug:.

I think the right thing here is to name this table `pendingAuthorizations`, the struct `pendingAuthorization`. Similarly for the next one: `authorizations` / `authorization`.